Update Firefox desktop pages post GitHub migration (#284)
flodolo/documentation: src/products/firefox_desktop/firefox_l10n_faqs.md

### I landed strings in mozilla-central, when are they going to be localized?

New string changes are extracted twice a day from `gecko-dev` (the git mirror of mozilla-central) into a repository called `firefox-l10n-source`, a unified repository that includes strings for all shipping versions of Firefox (nightly, beta, release). This repository is used as a buffer to avoid exposing potential issues to all locales working on Firefox.

> Typically once or twice a week, the content of the quarantine (the `update` branch) is pushed to the `main` branch which acts as the official repository, used by [Pontoon](https://pontoon.mozilla.org/projects/firefox/) as source reference. At this point, strings can be localized by the community.

Reviewer: ```suggestion
Typically once or twice a week, the quarantined content (in the `update` branch) is pushed to the `main` branch which acts as the official repository, used by [Pontoon](https://pontoon.mozilla.org/projects/firefox/) as source reference. At this point, strings can be localized by the community.
```

Strings need to be added to localization files in known locations within the mozilla-central tree. For all the technical details about these paths and the supported formats, see [this document](https://firefox-source-docs.mozilla.org/build/buildsystem/locales.html#exposing-strings).

Once strings land in mozilla-central, they will be exposed for localization in Pontoon typically within a week or sooner.

### Where can I find the localized strings?

> All shipping versions of Firefox are built from a single GitHub repository with a subdirectory for each locale called `firefox-l10n`, available [here](https://github.com/mozilla-l10n/firefox-l10n).

Reviewer: ```suggestion
All shipping versions of Firefox are built from a single GitHub repository called [firefox-l10n](https://github.com/mozilla-l10n/firefox-l10n), with a subdirectory for each locale.
```

It’s always possible to restore an old string that was removed from code, as long as the text remains the same, and the string is used exactly in the same context.

> It might also be possible to uplift the patch if the string is still available in the [firefox-l10n-source](https://github.com/mozilla-l10n/firefox-l10n-source). `firefox-l10n` acts as a single localization source for multiple builds, with a [superset](https://firefox-source-docs.mozilla.org/l10n/singlel10nsource.html) of strings from all [supported versions](https://github.com/mozilla-l10n/firefox-l10n-source/blob/main/.github/update-config.json#L2) of Firefox: nightly, beta, release, ESR.

Reviewer: ```suggestion
It might also be possible to uplift the patch if the string is still available in the [firefox-l10n-source](https://github.com/mozilla-l10n/firefox-l10n-source) repository. `firefox-l10n` acts as a single localization source for multiple builds, with a [superset](https://firefox-source-docs.mozilla.org/l10n/singlel10nsource.html) of strings from all [supported versions](https://github.com/mozilla-l10n/firefox-l10n-source/blob/main/.github/update-config.json#L2) of Firefox: nightly, beta, release, ESR.
```
